### PR TITLE
Adding team.md

### PR DESCRIPTION
**Pull request by luabida**
In order to create team.md I had to add the asset of flags that can be used to other pages and articles as well.

In this commit I also changed the about.md page that was written in html.

OBS: I couldn't fix the title in lowercase as index.md or even the blog page that has a fancy title (I don't get how this work).

### CODE REVIEW

**Review by luabida — COMMENTED** [review]
All tries (html, css inline, css style) of using an external link to the images failed. I had to keep the template using png archives but removed all unitilized flags, so we will have to download them manually.

File: pages/about.md [hunk 1]
```diff
@@ -1,27 +1,45 @@
-<h1>Open Science Labs</h1>
-<p>
-<strong>Open Science Labs</strong> es una iniciativa para divulgar prácticas de Ciencia Abierta a público en español.
-</p>
-<p>
+<!--
+
+.. title: About
```
Comment (xmnlab) [inline pages/about.md:2]: @luabida could you remove all the empty lines between each item pls?
it should looks like this https://github.com/OpenScienceLabs/opensciencelabs.github.io/pull/27/files#diff-3aeb9e99ca48f41cb79734a6bcaa49f82c42a2015f8b7839637b672613b7a534R2

**Review by xmnlab — DISMISSED** [review]
just a minor observation. other than that LGTM.

**Review by xmnlab — APPROVED** [review]
LGTM! thank you @luabida

**Pull request merged**